### ISSUE TITLE
ID3 tag support

### ISSUE THREAD

**Issue by ciros88** [aargirakis/BZRPlayer#38]

**Comment by ciros88** [comment]
think about displaying id3tag infos through the text scroller

**Comment by ciros88** [comment]
@aargirakis what do you think about?

**Comment by aargirakis** [comment]
Yes, could be cool, and I don't think it's a lot of work. But I guess we just show Artist - Title? Maybe Album and or year?

**Comment by ciros88** [comment]
Dunno, maybe all fields from the upper (of the info window) separated by a separator like hypen. Just try and see the result

**Comment by ciros88** [comment]
You could also try to put the field name followed by a colon first, maybe both rendered with a text size smaller then the field content text size
